test/grpc/error_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule GRPC.RPCErrorTest do
+	use ExUnit.Case
+
+  alias GRPC.RPCError
+
+  describe "is_rpc_error/2 guard clause" do
+    import GRPC.RPCError
```

Comment:
```suggestion
    import GRPC.RPCError, only: [is_rpc_error: 2]
```

for clarity :)
Also, I'd rather we imported after the `use` and before `alias`. And this file is missing `mix format`

Reply:
Filename is also incorrect. should be `rpc_error_test.exs`

Reply:
Fixed [f48e9d6](https://github.com/elixir-grpc/grpc/pull/314/commits/f48e9d6677db4caa481119b67ec5ec4aecd5d917)